src/serac/physics/heat_transfer.hpp

```diff
@@ -720,9 +806,17 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
   /// The temperature finite element state
   serac::FiniteElementState temperature_;
 
-  /// The adjoint temperature finite element state
+  /**
+   * @brief Rate of change in temperature at the current adjoint timestep
+   */
+  FiniteElementState temperature_rate_;
+
+  /// The adjoint temperature finite element states, the multiplier on the residual for a given timestep
   serac::FiniteElementState adjoint_temperature_;
 
+  // The total sensitivity of the qoi with respect to the start of the previos timestep's temperature
+  serac::FiniteElementDual _d_temperature_start_of_step_;
```

Comment:
Any reason for the leading underscore?

Reply:
Lack of creativity.  I was trying to invent some convention that leading _ means implicit derivative of the qoi.  But, that was before I noticed you called such things implicit sensitivities, or implicit derivatives (which is a term I like, though I hadn't heard before).  So I'll improve soon!